tfs/src/main/resources/hudson/plugins/tfs/TeamPRPushTrigger/config.jelly

```diff
@@ -0,0 +1,6 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">>
+  <f:entry title="Job Context" field="jobContext">
+    <f:textbox default="Jenkins CI build" />
```

Comment:
CI doesn't necessarily imply PR.  Can you change this to "Jenkins PR Build"?

Reply:
Sure, changed.